src/app/modules/players/players.controller.js

```diff
@@ -148,9 +149,21 @@ angular
        */
       function startMatch() {
         Match.reset();
+        localStorage.setItem('controlPlayer','0');
         $location.path('summary/round/1/player/' + Match.getNextPlayer(1).id);
       }
-
+      /**
+       * @ngdoc method
+       * @name PlayersController#loadMatch
+       * @kind function
+       * @methodOf app.players.controller:PlayersController
+       * @description
+       * Load a previus match and initializes 'controlPlayer' for the currentPlayer function on file sumrai.controller.js
+       */
+      function loadMatch() {
+        localStorage.setItem('controlPlayer','1');
```

Comment:
Non capisco le seguenti cose:
- il senso di numeri hardcodati incrementali.
- il significato di un valore come "controlPlayer": è poco esplicativo.

Reply:
Probabilmente questa non è una buona soluzione, ho voluto usare "controlPlayer" come appoggio per la funzione "currentPlayer()" in  sumray.controller.js, in modo da non perdere l'iformazione quando passo alla pagina del riepilogo della partita o aggiorno la pagina.

Reply:
Il problema non è dove lo memorizzi ma il fatto è che questa logica è leggibile solamente da te. Chiunque altro dovrebbe spendere del tempo a capire il perche di questa logica. Il naming e il giusto design pattern (nonche commenti) sono necessari per far spendere il minor tempo possibile, a chi legge questo codice, per capire cosa esso faccia. Chi legge potrebbe essere un altro sviluppatore o anche te fra qualche tempo, sono abbastanza sicuro che a distanza di tempo avresti difficolta anche te a capire cosa hai scritto tempo prima e perche.

Reply:
Va bene ho capito, cercherò di trovare una nuova soluzione quanto prima.